src/test/java/com/arpnetworking/metrics/mad/parsers/ProtobufV1ToRecordParserTest.java

```diff
@@ -1,4 +1,4 @@
-/*
+/**
```

Comment:
This should not be a Javadoc style comment. They were all removed in this code review:

https://github.com/ArpNetworking/metrics-aggregator-daemon/commit/41c5959e3451d030babbbd8c22f58dc71f0cd162#diff-ae01a70027d29879881cb1eb02c3b5c5

The checkstyle was updated. What error were you getting for this?